native_client/java/libdeepspeech/README.MD

```diff
@@ -0,0 +1,269 @@
+# DeepSpeech Java Bindings
+
+These are the Java bindings to use the DeepSpeech library from Java.
+
+## How to use
+
+Your Java application needs to load two native libraries: `libdeepspeech` and `libdeepspeech-jni`.
+
+__If you do not load these libraries, the invocation of the native functions will throw an `UnsatisfiedLinkError`.__
+
+Both are shared library which can be found in the DeepSpeech native client releases. 
+Make sure that binaries for all OSs and configurations _(OS, CPU / Cuda)_ your application will run on are included in your application. _(eg. classpath, installation directory)_.
+
+> See [DeepSpeech releases]
+
+Loading the native necessary library from your Java Application is done using `DeepSpeechLibraryConfig`.
+
+There are two main types of library configurations:
+
+* Desktop
+    * CPU
+    * CUDA
+* Android
+
+On a desktop pc (except OSX for now) cuda acceleration is supported. This requires a compatible GPU and Driver.
+
+### The Desktop CPU configuration
+
+```java
+DeepSpeechLibraryConfig config = new DeepSpeechLibraryConfig.Desktop.CPU(
```

Comment:
The API is here to abstract complexity, not to expose it.

Reply:
The library loading aspect should be exposed. You want to have control in this aspect when you are supporting more than just android. On Android you don't need to have access to the library loading part. On other applications you don't want a System.loadLibrary() call hard-coded in the static block of a class. Whats wrong with not hard-coding?

Reply:
And then you endup with useless boiler plate code. Using `libdeepspeech.so` + `libdeepspeech-jni.so`, I have absolutely no problem with hard-coded `System.loadLibrary()` call.